charts/kong/values.yaml

```diff
@@ -102,6 +127,9 @@ proxy:
     # overrideServiceTargetPort: 8000
```

Comment:
Any reason this port is not carried over to other services?
For folks who terminate TLS at the LB layer, they need this, else things usually break.

We recommend the Ingress Controller setup but there will be always a set of users who will need to terminate TLS at ELB.

Reply:
It's only implemented for the proxy Service template. Consolidating those is a long-term goal, but out of scope for this.

Reply:
Okay, tracking this internally.